### DISCUSSION TITLE
Version 3.2.0 Crashes

### DISCUSSION THREAD

**Discussion by klafredo** [martinet101/ElevenClock#561] [General]
Is anyone else seeing this.  With version 3.0.2 if I right click on it in the taskbar it crashes every time.  The only way to get to the preferences is to use the shortcut.

**Comment by marticliment** [comment]
can you please share your log and a video of the crash?

**Comment by klafredo** [comment]
Attached are the dump files and logs.

**Reply by marticliment** [reply]
I don't think github accept attached files as answers, because I can't see them...

**Comment by klafredo** [comment]
What’s the best way for me to get them to you?

**Comment by klafredo** [comment]
Here is the log…but not much in it

---------------------------------------------------------------------------------------------------

   ElevenClock's 3.2.0 (v3.2) log: Select all the text and hit Ctrl+C to copy it
   All modules loaded successfully and sys.stdout patched correctly, starting main script
   Translator function set language to "en"

---------------------------------------------------------------------------------------------------

Log legend:
🔵: Verbose
🟢: Information
🟡: Warning
🟠: Handled unexpected exception
🔴: Unhandled unexpected exception
🟣: Handled expected exception

🟡 This is the primary screen and is set to be skipped
🔵 Initializing clock 0...
🟡 Not using taskbar background color
🔵 Using bg color: 17,39,64,100.0
🟡 Custom height being used!
🔵 Monitor geometry: PyQt5.QtCore.QRect(-1920, 2, 1920, 1080)
🟢 Taskbar at top
🟢 Clock on the right
🟡 X offset being used!
🟡 Y offset being used!
🟢 Using qt's default positioning system
🔵 Clock geometry: PyQt5.QtCore.QRect(-190, -1, 200, 43)
🔵 Font families: ['Segoe UI Variable Display', 'sans-serif']
🔵 Font size: 8.0
🟢 Using white text (dark mode)
🔵 Restart loop: 1
🔵 Full screen rect:  (-1920, 2, 0, 1082)
🔵 Show/hide loop started with parameters: HideonFS:True, NotHideOnTB:False, HideOnRDP:False, ClockOn1Mon:False, NefWSMethod:True, DisableNotifications:True, legacyFullScreenMethod:False
🟡 Desktop button enabled
🔵 Initializing clock 1...
🟡 Not using taskbar background color
🔵 Using bg color: 17,39,64,100.0
🟡 Custom height being used!
🔵 Monitor geometry: PyQt5.QtCore.QRect(1920, 0, 1920, 1080)
🟢 Taskbar at top
🟢 Clock on the right
🟡 X offset being used!
🟡 Y offset being used!
🟢 Using qt's default positioning system
🔵 Clock geometry: PyQt5.QtCore.QRect(3650, -3, 200, 43)
🔵 Font families: ['Segoe UI Variable Display', 'sans-serif']
🔵 Font size: 8.0
🟢 Using white text (dark mode)
🔵 Full screen rect:  (1920, 0, 3840, 1080)
🔵 Show/hide loop started with parameters: HideonFS:True, NotHideOnTB:False, HideOnRDP:False, ClockOn1Mon:False, NefWSMethod:True, DisableNotifications:True, legacyFullScreenMethod:False
🟡 Desktop button enabled
🟢 Loaded clocks in 3.610666513442993
🔵 System tray icon enabled
🔵 Starting update check
🔵 Low cpu mode is set to False. DisableNotifications is set to False
🟢 Loaded everything in 5.801712989807129
🔵 Version URL: https://versions.somepythonthings.tk/versions/elevenclock.ver
🟢 Updates not found
🔵 Announcement URL: http://www.somepythonthings.tk/resources/elevenclock.announcement
🔵 Image URL: https://www.somepythonthings.tk/resources/we-are-on-twitter.webp
🔵 Copying log to the clipboard...
🟢 Log copied to the clipboard successfully!
🔵 Starting update check
🔵 Version URL: https://versions.somepythonthings.tk/versions/elevenclock.ver
🟢 Updates not found
🔵 Copying log to the clipboard...

From: Ken J. Lafredo
Sent: Thursday, February 3, 2022 10:52 AM
To: martinet101/ElevenClock ***@***.***>; martinet101/ElevenClock ***@***.***>
Cc: Author ***@***.***>
Subject: RE: [martinet101/ElevenClock] Version 3.2.0 Crashes (Discussion #561)

What’s the best way for me to get them to you?

**Comment by marticliment** [comment]
If you maintain a settings window open and you right click the clock, what happens?

**Comment by klafredo** [comment]
Yes, with the settings window already open then right clicking on the app in the taskbar it still crashes.

**Reply by marticliment** [reply]
But the settings close?

**Comment by klafredo** [comment]
Yes, after the crash it is gone.

**Comment by marticliment** [comment]
Ok... Can you please delete the .elevenclock folder placed in your user directory?

**Comment by klafredo** [comment]
Done..kind of fixed it…but not really.
So when I started it up the clock showed up at the bottom right of my screen.  I could right click on it and the menu popped up and no crash.
I then started to put back all of my settings.  The first thing I did was set the “Force clock to top of the screen” (that is where I have my taskbar).  I then right clicked on the clock and it crashed.

**Reply by marticliment** [reply]
Ok, then can you please compress the .elevenclock folder as you have it now and send it to me? Because there must be some kind of unhandled error. Thx!

**Comment by klafredo** [comment]
How do I send it?  I thought GitHub does not allow attachments.

**Reply by marticliment** [reply]
You can send it to me at marticlilop@gmail.com

**Reply by marticliment** [reply]
Ok, i've recieved them, i'll take a look and i'll let you know

**Comment by marticliment** [comment]
All right, I fixed the issue, it resulted that it was getting negative mouse positions and then it crashed. Fixed, coming with the next version

**Comment by klafredo** [comment]
Thanks!

**Comment by klafredo** [comment]
ElevenClock 3.2.1 fixed the crash...but right clicking still does not show the menu.  Unless the menu is being drawn up (and off screen) and so can't be seen, just a guess.

**Reply by marticliment** [reply]
In theory it should be drawn correctly. Anyway, you can access the menu through the systwm tray icon

**Reply by klafredo** [reply]
I was able to fix it.
